src/main/java/Duke.java

```diff
@@ -1,10 +1,74 @@
+import java.util.Scanner;
+
+/**
+ * Represents the main class for the application Duke.
+ */
 public class Duke {
+
+    /**
+     * Main entry point of the Duke application.
+     * Initializes the application and starts the interaction with the user.
+     *
+     * @param args Command line arguments.
+     */
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        // Initialize greeting logo and print welcome message
+        String logo =
+                "   J     A    SSSS    OOO   N   N \n"
+                        + "   J    A A   S      O   O  NN  N \n"
+                        + "   J   AAAAA   SSS   O   O  N N N \n"
+                        + "J  J  A     A     S  O   O  N  NN \n"
+                        + " JJJ  A     A  SSSS   OOO   N   N \n" +
+                        "                 \n";
+        System.out.println(logo + " Eyy wassup I'm Jason\r\n"
+                + " What can I do for you?\r\n"
+        );
+
+        Scanner scanner = new Scanner(System.in);
+        String input;
+        Task[] list = new Task[100];
```

Comment:
Avoid having magic number of 100, use named constant such as static final int MAX_TASKS instead